[2022-02-02 21:09:56,871] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-02 21:09:56,939] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-02 21:09:56,942] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:09:56,943] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 21:09:56,951] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:09:57,038] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-02 06:00:00+00:00
[2022-02-02 21:09:57,047] {standard_task_runner.py:52} INFO - Started process 448 to run task
[2022-02-02 21:09:57,053] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpcmqyvb8q', '--error-file', '/tmp/tmppql6md5f']
[2022-02-02 21:09:57,055] {standard_task_runner.py:77} INFO - Job 316: Subtask local_to_gcs_task
[2022-02-02 21:09:57,255] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:09:57,703] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 21:09:57,813] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-02 21:25:43,860] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:25:43,895] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20190102T060000, start_date=20220202T210956, end_date=20220202T212543
[2022-02-02 21:25:43,947] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:25:44,051] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 21:16:44,061] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-09 21:16:44,100] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-09 21:16:44,100] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 21:16:44,101] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-09 21:16:44,101] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 21:16:44,144] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-02 06:00:00+00:00
[2022-02-09 21:16:44,151] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2022-02-09 21:16:44,156] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '437', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmp_d5i370x', '--error-file', '/tmp/tmpvp73s3fp']
[2022-02-09 21:16:44,158] {standard_task_runner.py:77} INFO - Job 437: Subtask local_to_gcs_task
[2022-02-09 21:16:44,317] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host 9fc7f13f453f
[2022-02-09 21:16:44,453] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 21:16:44,515] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-09 21:31:23,623] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 21:31:23,669] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20190102T060000, start_date=20220209T211644, end_date=20220209T213123
[2022-02-09 21:31:23,725] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 21:31:23,816] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 22:32:58,490] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-09 22:32:58,526] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-09 22:32:58,526] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 22:32:58,527] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-09 22:32:58,527] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 22:32:58,568] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-02 06:00:00+00:00
[2022-02-09 22:32:58,574] {standard_task_runner.py:52} INFO - Started process 312 to run task
[2022-02-09 22:32:58,580] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '454', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmpxx7870x4', '--error-file', '/tmp/tmp3bpfed_h']
[2022-02-09 22:32:58,582] {standard_task_runner.py:77} INFO - Job 454: Subtask local_to_gcs_task
[2022-02-09 22:32:58,722] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host 0fd4ea805f5e
[2022-02-09 22:32:58,846] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 22:32:58,902] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-09 22:47:49,589] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 22:47:49,622] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20190102T060000, start_date=20220209T223258, end_date=20220209T224749
[2022-02-09 22:47:49,718] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 22:47:49,806] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 01:45:07,171] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-19 01:45:07,218] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-19 01:45:07,219] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 01:45:07,219] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 01:45:07,220] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 01:45:07,264] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-02 06:00:00+00:00
[2022-02-19 01:45:07,272] {standard_task_runner.py:52} INFO - Started process 3743 to run task
[2022-02-19 01:45:07,278] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '619', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmply282dm_', '--error-file', '/tmp/tmp8s2d458h']
[2022-02-19 01:45:07,280] {standard_task_runner.py:77} INFO - Job 619: Subtask local_to_gcs_task
[2022-02-19 01:45:07,438] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host 312f1fe573aa
[2022-02-19 01:45:07,589] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 01:45:07,654] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-19 02:01:01,693] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 02:01:01,737] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20190102T060000, start_date=20220219T014507, end_date=20220219T020101
[2022-02-19 02:01:01,827] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 02:01:01,931] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 11:44:43,476] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-19 11:44:43,514] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [queued]>
[2022-02-19 11:44:43,515] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 11:44:43,515] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 11:44:43,515] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 11:44:43,569] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-01-02 06:00:00+00:00
[2022-02-19 11:44:43,577] {standard_task_runner.py:52} INFO - Started process 4590 to run task
[2022-02-19 11:44:43,582] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question3_FHV', 'local_to_gcs_task', 'scheduled__2019-01-02T06:00:00+00:00', '--job-id', '989', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions3.py', '--cfg-path', '/tmp/tmps11zjy_q', '--error-file', '/tmp/tmpkfy_8lnz']
[2022-02-19 11:44:43,585] {standard_task_runner.py:77} INFO - Job 989: Subtask local_to_gcs_task
[2022-02-19 11:44:43,733] {logging_mixin.py:109} INFO - Running <TaskInstance: Question3_FHV.local_to_gcs_task scheduled__2019-01-02T06:00:00+00:00 [running]> on host f81f14254480
[2022-02-19 11:44:43,898] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 11:44:43,998] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question3_FHV
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-02T06:00:00+00:00
[2022-02-19 12:00:41,721] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 12:00:41,764] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question3_FHV, task_id=local_to_gcs_task, execution_date=20190102T060000, start_date=20220219T114443, end_date=20220219T120041
[2022-02-19 12:00:41,843] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 12:00:41,949] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
